.gitignore

```diff
@@ -1,225 +0,0 @@
-## Ignore Visual Studio temporary files, build results, and
```

Comment:
Any reason why the .gitignore is deleted?

Reply:
This should be by mistake when i did a cherry pick. I will put it back in